sway-lib-std/src/u128.sw

```diff
@@ -95,16 +95,16 @@ impl u64 {
 
         asm(sum, overflow, left: self, right: right, result_ptr: result) {
             // Add left and right.
-            add sum left right;
+            add  sum left right;
```

Comment:
the extra spaces in this and the following `asm` blocks don't look correct.

Reply:
IIRC they're adding padding to make each asm opcode take up 4 characters for alignment. However, I thought that was because none of the opcodes were longer than 4 characters, but I see `right` is 5 chars. I don't see `left` or `right` keywords in the [docs](https://specs.fuel.network/master/fuel-vm/instruction-set.html).

Reply:
They are not. See their declaration on line 96.